jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/LambdaRunConfigurationBase.kt

```diff
@@ -49,8 +49,12 @@ abstract class LambdaRunConfigurationBase<T : BaseLambdaOptions>(
 
     protected fun checkInput() {
         inputSource()?.let {
-            if (isUsingInputFile() && FileUtil.exists(it)) {
-                return
+            if (isUsingInputFile()) {
+                if (FileUtil.exists(it)) {
+                    return
+                }
+            } else {
+                return // Using input text and its not null
             }
```

Comment:
A more compact version:
```kotlin
if (!isUsingInputFile() || FileUtil.exists(it)) {
    return
}
```
I personally think it is easier to understand, but I'm ok with the original version.

Reply:
Done